src/embed/index.js

```diff
@@ -0,0 +1,176 @@
+import './embed.less';
+import {
+  Container,
+  HelpPlugin,
+  CaptionsPlugin,
+  SoundPlugin,
+  PausePlugin,
+  UserDataPlugin
+} from 'springroll-container';
+import '../libs/jquery.mobile.custom';
+import '../plugins/jquery-menuToggle';
+
+const container = new Container('#appContainer');
+const frame = $('#frame');
+const captionsToggle = $('#captionsToggle');
+const soundToggle = $('#soundToggle');
+const title = $('#appTitle');
+captionsToggle.hide();
+soundToggle.hide();
+
+const captions = new CaptionsPlugin('#captionsButton');
+
+container.uses(new UserDataPlugin());
+container.uses(new HelpPlugin('#helpButton'));
+container.uses(captions);
+container.uses(new PausePlugin('#pauseButton, #resumeButton'));
+container.uses(
+  new SoundPlugin({
+    voButton: '#voButton',
+    soundButton: '#soundButton',
+    sfxButton: '#sfxButton',
+    musicButton: '#musicButton',
+    voSlider: '#voiceRange',
+    musicSlider: '#musicRange',
+    sfxSlider: '#sfxRange',
+    soundSlider: '#mainVolume'
+  })
+);
```

Comment:
Looks like the uses function is no longer available on the SpringRollContainer. You'll have to switch to the new syntax:

```javascript
new springroll.Container('selector', {
  plugins: [...]
});